solidity/test/token_stake/TestTopUps.js

```diff
@@ -763,6 +793,21 @@ describe('TokenStaking/TopUps', () => {
       )
       expect(currentStake).to.eq.BN(delegatedAmount.muln(4))
     })
+
+    it("can be done with less than the minimum stake", async () => {
+      const topUpAmount = 11
```

Comment:
It may be better to set this value relative to the minimum stake.
```suggestion
      const topUpAmount = (await tokenStaking.minimumStake()).subn(1)
```

Reply:
Please see https://github.com/keep-network/keep-core/pull/1974/commits/81619c70a48b3f53828a3f0894ddfbb78e00343e